multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java

```diff
@@ -20,17 +20,54 @@
 import com.xebialabs.restito.server.StubServer;
 import org.glassfish.grizzly.http.util.HttpStatus;
 import org.hamcrest.Matchers;
+import org.json.JSONObject;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 import org.springframework.web.context.WebApplicationContext;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ContextConfiguration(initializers = MultiIntStoreIntegrationTest.Initializer.class)
+@EnableConfigurationProperties
+public class MultiIntStoreIntegrationTest {
+
+  @ClassRule
+  public static final GenericContainer solr =
```

Comment:
✏️ what was the reasoning to move this outside the abstract class?

Reply:
I wanted to be really clear about the difference between `MultiIntStoreIntegrationTest` and `MultiIntStoreMockSolrIntegrationTest`. I was thinking it'd be confusing why `MultiIntStoreMockSolrIntegrationTest` didn't also extend `MultiIntStoreIntegrationTestContainers`.